src/packages/@ncigdc/modern_components/ClinicalAnalysis/ClinicalVariableCard/index.js

```diff
@@ -1306,10 +1319,19 @@ const ClinicalVariableCard: React.ComponentType<IVariableCardProps> = ({
                           updateClinicalAnalysisVariable({
                             fieldName,
                             id,
-                            value: 0,
+                            value: {},
                             variableKey: 'continuousCustomInterval',
                           })
                         ));
+                      variable.plotTypes === 'continuous' && (
```

Comment:
`variable.plotTypes === 'continuous' &&` removed in the next ticket/PR

you can only open/save this modal if you're on `continuous` already